apps/web/src/contexts/globalLayout/GlobalNavbar/CollectionNavbar/CollectionNavbar.tsx

```diff
@@ -91,30 +87,27 @@ export function CollectionNavbar({ queryRef, username, collectionId }: Collectio
   return (
     <StandardNavbarContainer>
       <NavbarLeftContent>
-        <ProfileDropdown
-          queryRef={query}
-          rightContent={
-            isMobile ? null : (
-              <RightContentWrapper shrink gap={4}>
-                {query.userByUsername && (
-                  <NavActionFollow userRef={query.userByUsername} queryRef={query} />
-                )}
-
-                <SlashText>/</SlashText>
-
-                <Link href={galleryRoute} legacyBehavior>
-                  <BreadcrumbLink href={route(galleryRoute)}>{galleryName}</BreadcrumbLink>
-                </Link>
-
-                <SlashText>/</SlashText>
-
-                <CollectionNameText title={unescapedCollectionName}>
-                  {unescapedCollectionName}
-                </CollectionNameText>
-              </RightContentWrapper>
-            )
-          }
-        />
+        <Wrapper gap={4} align="center">
```

Comment:
The change here is removing `ProfileDropdown`, so we can now directly render what used to be `rightContent`